### PR TITLE
tile data - fix language and number notation

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Video_Display.md [hunk 1]
```diff
@@ -439,61 +439,61 @@ Speed Mode).
...
-Sprites always use 8000 addressing, but the BG and Window can use either
+Sprites always use "$8000 addressing", but the BG and Window can use either
 mode, controlled by [LCDC bit
 4](#lcdc-4-bg-window-tile-data-select).
 
-Each Tile occupies 16 bytes, where each 2 bytes represent a line:
+Each tile occupies 16 bytes, where each line is represented by 2 bytes:
 
 ```
 Byte 0-1  First Line (Upper 8 pixels)
```
Comment (ISSOtm) [inline content/Video_Display.md:474]: Maybe "topmost"?

File: content/Video_Display.md [hunk 2]
```diff
@@ -439,61 +439,61 @@ Speed Mode).
...
-Bit 0 the rightmost. For example : let's say you have \$57 \$36 (in
-this order in memory). To obtain the color index for the leftmost pixel,
-you take bit 7 of both bytes : 0, and 0. Thus the index is 00b = 0. For
-the second pixel, repeat with bit 6 : 1, and 0. Thus the index is 01b =
-1 (remember to flip the order of the bits !). If you repeat the
-operation you'll find that the indexes for the 8 pixels are 0 1 2 3 0 3
+For each line, the first byte defines the least significant bit of the
+color ID of each pixel, and the second byte defines the upper bits
+of the color IDs. In either case, bit 7 is the leftmost pixel, and
+bit 0 the rightmost. For example : let's say you have \$57 \$36 (in
```
Comment (ISSOtm) [inline content/Video_Display.md:482]: No spaces before colons, please

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Video_Display.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

File: content/Video_Display.md (hunk 2 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Comment by ISSOtm** [comment]
Needs rebase + extra review

**Comment by AlvaroBurnett** [comment]
Rebased

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**